record_store.gemspec

```diff
@@ -22,11 +22,14 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'thor'
   spec.add_runtime_dependency 'activesupport', '~> 4.2'
   spec.add_runtime_dependency 'activemodel', '~> 4.2'
-  spec.add_runtime_dependency 'fog'
+  spec.add_runtime_dependency 'ejson'
+
+  spec.add_runtime_dependency 'fog', '>= 1.33.0'
```

Comment:
Bundler would revert the `fog` gem to 0.7.2 without specifying a version here. I'm not sure why.